rest-server/src/main/resources/static/index.html

```diff
@@ -943,7 +943,7 @@ <h3 class="text-lg leading-6 font-medium text-gray-900">
                                         <div class="p-3 border rounded px-5 py-6 "
                                              :class="components.database.size==='basic' ? 'bg-blue-900 text-white' : 'border-gray-300 bg-white text-black-500 hover:bg-blue-100'"
                                              @click="components.database.size = 'basic'"
-                                             x-show="components.database.type!=='SQL_SERVER'"><span class="text-2xl">Basic</span>
+                                             x-show="components.database.type==='MYSQL'"><span class="text-2xl">Basic</span>
```

Comment:
That change was because this Basic div is only for MySQL now, it used to be for both MYSQL and POSTGRES, but there were enough differences in the low tier (name, SLA) that I made a brand new div.